source/controllers/CurrentTimeController.mc

```diff
@@ -2,16 +2,28 @@ import Toybox.Application;
 import Toybox.Lang;
 import Toybox.Time;
 
-class CurrentTimeController {
+class CurrentTimeController extends BaseController {
+  function initialize() {
+    BaseController.initialize();
+  }
+
+  public function getAngle() as Numeric {
```

Comment:
If an ArcComponent uses this controller, the arc will display the "progress" of a 24hr day. So at noon, the arc will be at the 6 o-clock position, while at midnight it is at 12 o-clock.